src/reconstruct/plm.cpp

```diff
@@ -28,226 +28,234 @@
 
 namespace parthenon {
 
-//----------------------------------------------------------------------------------------
-//! \fn Reconstruction::PiecewiseLinearX1()
-//  \brief
+namespace {
 
-void Reconstruction::PiecewiseLinearX1(const int k, const int j, const int il,
-                                       const int iu, const ParArrayND<Real> &q,
-                                       ParArrayND<Real> &ql, ParArrayND<Real> &qr) {
-  auto pmb = GetBlockPointer();
-  auto &coords = pmb->coords;
-  // set work arrays to shallow copies of scratch arrays
-  ParArrayND<Real> &qc = scr1_ni_, &dql = scr2_ni_, &dqr = scr3_ni_, &dqm = scr4_ni_;
-  const int nu = q.GetDim(4) - 1;
+enum class Dimension { X1, X2, X3 };
```

Comment:
This already exists as `X1DIR`, `X2DIR`, `X3DIR` in `defs.hpp`. I think we should use that one for this kind of thing whenever possible.

I know it's an enum there, not an enum class, but I don't like introducing new local notation. I think we should use the same notation code-wide.